database/sqlite/sqlite_functions.h

```diff
@@ -63,6 +63,7 @@ extern int find_uuid_type(uuid_t *uuid);
 extern void sql_rrdset2json(RRDHOST *host, BUFFER *wb);
 
 extern RRDHOST *sql_create_host_by_uuid(char *guid);
+extern int prepare_statement(sqlite3 *database, char *query, sqlite3_stmt **statement);
```

Comment:
in sqlite_functions.c you have removed static from 2 functions:
- execute_insert
- prepare_statement
but in sqlite_functions.h you added only
- prepare_statement

Reply:
I will fix it  (now it exists in sqlite_aclk_chart.h)